### PR TITLE
Protected delete

### PR DESCRIPTION
**Pull request by Kmmontes**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -33,23 +59,49 @@ class App extends Component {
...
+            />
+
+
+          <Route path="/gooseyindex" component={GooseyIndex}/>
+
           <Route path="/gooseynew" component={GooseyNew} current_user={current_user}/>
-          <Route
-            path="/gooseyshow/:id"
+
+          {/* <Route path="/gooseyshow/:id"
```
Comment (elysemontano) [inline app/javascript/components/App.js:82]: Are you using this?  If not can it be removed?

File: app/javascript/components/components/DeleteGooseyButton.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    .catch(errors => {
+      console.log("delete errors:", errors)
+    })
+  }
+
+
+
+  render() {
+    return (
+        <NavLink to= "/mygooseylist">
```
Comment (elysemontano) [inline app/javascript/components/components/DeleteGooseyButton.js:40]: Indentation

File: app/javascript/components/components/EditGooseyButton.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
```
Comment (elysemontano) [inline app/javascript/components/components/EditGooseyButton.js:1]: Double check your indentation on this component.

File: app/javascript/components/pages/GooseyEdit.js [hunk 4]
```diff
@@ -1,11 +1,162 @@
...
+import { Redirect } from 'react-router-dom'
+import DeleteGooseyButton from '../components/DeleteGooseyButton'
+import EditGooseyButton from '../components/EditGooseyButton'
 
 export default class GooseyEdit extends Component {
+    constructor(props) {
+        super(props)
+        this.state ={
+          editListing: {
+            id: 23,
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyEdit.js:12]: This should not be hard coded.  Try using this.props.listing.id

File: app/javascript/components/pages/GooseyEdit.js [hunk 5]
```diff
@@ -1,11 +1,162 @@
...
+        })
+        .catch(errors => {
+          console.log("create errors:", errors)
+        })
+}
+
+
+    
     render() {
+      console.log(this.props)
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyEdit.js:56]: Please remove these console logs.

File: app/javascript/components/pages/GooseyEdit.js [hunk 6]
```diff
@@ -1,11 +1,162 @@
...
+// }
+
+
+
+
+
+
+
+
+
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyEdit.js:162]: Can we delete this?

File: app/javascript/components/pages/GooseyShow.js [hunk 7]
```diff
@@ -2,10 +2,28 @@ import React, { Component } from 'react'
...
+                <img src={this.props.listing.image} width="200" height="200" />
+                  {/* <NavLink to={`/gooseyedit/${this.props.listing.id}`}>
+                    <Button>Edit Listing Profile</Button>
+                  </NavLink>
+
+                  <NavLink to= "/listingindex">
+                    <Button onClick={ () => this.props.deleteListing(this.props.listing.id)}> Delete Food Profile </Button>
+                  </NavLink>
+                 */}
+                  </div>}
```
Comment (elysemontano) [inline app/javascript/components/pages/GooseyShow.js:25]: Please double-check your indentation.

File: app/javascript/components/pages/MyGooseyList.js [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+import { NavLink } from 'react-router-dom'
+import DeleteGooseyButton from '../components/DeleteGooseyButton';
+import EditGooseyButton from '../components/EditGooseyButton';
+
+export default class MyGooseyList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      myListings: [],
+      id: 1,
```
Comment (elysemontano) [inline app/javascript/components/pages/MyGooseyList.js:12]: This should be dynamic and not hardcoded.

File: app/javascript/components/pages/MyGooseyList.js [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+                    <img src={listing.location_image} width="200" height="200" />
+                    <p className="my-card-button">                  
+
+                    <NavLink to={`/gooseyedit/${listing.id}`}>
+                     <Button>Edit Listing </Button>
+                   </NavLink>
+
+                  <DeleteGooseyButton listing_id={listing.id} />
+                  </p>
+            </div>
```
Comment (elysemontano) [inline app/javascript/components/pages/MyGooseyList.js:94]: Check indentation

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left some comments.  Please remove any extra console logs and make sure all IDs in state are dynamic and not hard-coded.

**Review by triciasykes — APPROVED** [review]

**Review by triciasykes — APPROVED** [review]

**Pull request merged**